vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -176,7 +176,7 @@ def save_guest_devices_inventory(hardware, hashes)
       end
     end
 
-    deletes = hardware.guest_devices.find(:all, :conditions => {:device_type => ['ethernet', 'storage']}).dup
+    deletes = hardware.guest_devices.where(:device_type => ["ethernet", "storage"]).to_a.dup
```

Comment:
Is the dup even necessary anymore?  I thought to_a returned a new Array, or does it return a reference to the cached Array inside of the AR::Relation.